### ISSUE TITLE
The program can't start because Qt5Widgetsd.dll is missing from your computer

### ISSUE THREAD

**Issue by StoyanDimitrov** [martinrotter/rssguard#301]
#### Brief description of the issue.
Well, it is missing some dependency.

#### How to reproduce the bug?

1. Install the application or unzip a release archive
2. Double click on the application executable

#### What was the expected result?

The application should start

#### What actually happened?

The error is shown (this error is from the `nowebengine` zip the other ones is below)
```
---------------------------
rssguard.exe - System Error
---------------------------
The program can't start because Qt5Widgetsd.dll is missing from your computer. Try reinstalling the program to fix this problem. 
---------------------------
OK   
---------------------------
```

```
---------------------------
rssguard.exe - System Error
---------------------------
The program can't start because Qt5WebEngineWidgetsd.dll is missing from your computer. Try reinstalling the program to fix this problem. 
---------------------------
OK   
---------------------------
```

#### Other information (logs, see Wiki)

Tried both zips, and the `nowebengine` install. The OS is Windows 7.

**Comment by mfrsousa** [comment]
+1

**Comment by martinrotter** [comment]
Oh shit, will check it out. Will delete 3.8.1 and fix, thank you guys.

**Comment by martinrotter** [comment]
You are right, there is a problem. Will fix.

**Comment by martinrotter** [comment]
Should be fixed now with 3.8.2.
